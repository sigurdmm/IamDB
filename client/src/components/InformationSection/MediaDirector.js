import PropTypes from 'prop-types';
import React from 'react';

const MediaDirector = ({ director }) => <h3>{`ðŸŽ¬: ${director || 'unknown'}`}</h3>;

MediaDirector.propTypes = {
  director: PropTypes.string,
};

export default MediaDirector;
