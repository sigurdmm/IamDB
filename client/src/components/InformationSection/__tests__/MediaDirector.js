import React from 'react';
import { render } from 'enzyme/build';
import MediaDirector from '../MediaDirector';

const mockedDirectors = {
  existingDirector: 'Existing Director',
};

describe('<MediaDirector/>', () => {
  it('should render director when director is set', () => {
    const wrapper = render(<MediaDirector director={mockedDirectors.existingDirector}/>);

    expect(wrapper).toMatchSnapshot();
  });
});
