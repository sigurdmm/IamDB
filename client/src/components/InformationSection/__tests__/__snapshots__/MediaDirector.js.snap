// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaDirector/> should render director when director is set 1`] = `
<h3>
  ðŸŽ¬: Existing Director
</h3>
`;
